cxx17.cmake

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2013-2017, Ruslan Baratov
+# Copyright (c) 2017, Richard Hodges
+# All rights reserved.
+
+if(DEFINED POLLY_CXX17_CMAKE_)
```

Comment:
Please add new toolchains to `bin/details/toolchain_table.py`, see CONTRIBUTING:
* https://github.com/ruslo/polly/blob/master/CONTRIBUTING.md